web/source/osk/visualKeyboard.ts

```diff
@@ -1461,19 +1488,24 @@ namespace com.keyman.osk {
 
       let kbdObj = new VisualKeyboard(PKbd, device.coreSpec, device.coreSpec, true);
 
-      // The 'documentation' format uses the base element's child as the actual display base.
-      // Since there's no backing kmw-osk-frame, we do need the static-class kmw-osk-inner-frame
-      // to perform background styling on our behalf.  We'll trust the actual, live keyboard rules
-      // for the other elements, which in turn needs the non-static variant of the CSS rules.
-      kbdObj.layerGroup.element.className = kbdObj.kbdDiv.className + ' ' + device.formFactor
-        + '-static ' + device.OS.toLowerCase();
+      kbdObj.layerGroup.element.className = kbdObj.kbdDiv.className; // may contain multiple classes
+      kbdObj.layerGroup.element.classList.add(device.formFactor + '-static');
 
       let kbd = kbdObj.kbdDiv.childNodes[0] as HTMLDivElement; // Gets the layer group.
 
+      // Models CSS classes hosted on the OSKView in normal operation.  We can't do this on the main
+      // layer-group element because of the CSS rule structure for keyboard styling.
+      //
+      // For example, `.ios .kmw-keyboard-sil_cameroon_azerty` requires the element with the keyboard
+      // ID to be in a child of an element with the .ios class.
+      let classWrapper = document.createElement('div');
+      classWrapper.classList.add(device.OS.toLowerCase());
+      classWrapper.classList.add(device.formFactor.toLowerCase());
```

Comment:
```suggestion
      classWrapper.classList.add(device.OS.toLowerCase(), device.formFactor);
```

We are inconsistent between `OS` and `formFactor` already but let's at least keep `formFactor` consistent with its `-static` class name?